test/multiverse/suites/mongo/Envfile

```diff
@@ -13,19 +13,15 @@ MONGO_VERSIONS_RUBY25 = ['2.15.1',
                          '2.12.0']
 
 # Mongo versions to test with all CRubies
-MONGO_VERSIONS_ALL_RUBIES = ['2.9.0',
-                             '2.7.0',
-                             '2.5.0',
-                             '2.4.0',
-                             '2.1.0',
-                             '2.0.0',
-                             '1.10.0',
-                             ['1.9.2', 2.6],
-                             ['1.8.6', 2.6],
-                             '1.7.1']
-
-# Include bson_ext for older Mongo gem versions <= 1.10.0
-BSON_EXT_NEEDED_BEFORE_VERSION = Gem::Version::new("1.10.0")
+MONGO_VERSIONS_OLDER_RUBIES = ['2.9.0',
+                               '2.7.0',
+                               '2.5.0',
+                               '2.4.0',
+                               '2.1.0',
+                               '2.0.0',
+                               ['1.9.2', 2.6],
+                               ['1.8.6', 2.6],
```

Comment:
What does 2.6 mean here?

Reply:
Wow, I'd forgotten about this one from the last time I was in this file. It represents the "last supported server version" for the given gem version, and will prevent the testing of the gem if the server version is too new. But weirdly we don't apply that same constraint to the oldest gem version, `'1.7.1'`. I'll see if I can clean this up while we're in here. Depending on how old these oldest 1.x versions are, I might just drop them entirely as part of our gem EOL efforts.

Reply:
Addressed with [ecdc7b8](https://github.com/newrelic/newrelic-ruby-agent/pull/1067/commits/ecdc7b89b1a30684e6a98d54b87f2c267904ad56)